api/graphite.go

```diff
@@ -695,11 +695,22 @@ func (s *Server) executePlan(ctx context.Context, orgId uint32, plan expr.Plan)
 	reqs := NewReqMap()
 	metaTagEnrichmentData := make(map[string]tagquery.Tags)
 
+	// Map identical series expressions to reduce round trips
+	resolveSeriesRequests := make(map[expr.Req][]*expr.Req)
```

Comment:
Is there a good reason why we need this to be a pointer to `expr.Req`, as opposed to just copying the value? If not I'd copy it by value